src/content/docs/new-relic-solutions/new-relic-one/build-nr-apps/advanced-config.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Enable advanced configurations for your Nerdpack"
+tags:
+    - New Relic 
+    - Developer account
+    - API key
+    - New Relic One CLI
+metaDescription: Enable advanced configurations for your Nerdpack.
+freshnessValidatedDate: 2024-05-27
```

Comment:
```suggestion
freshnessValidatedDate: never
```

This field should be "never" for every doc migrated. We'll need to revisit all of this content sooner rather than later.